packages/common/src/ui/components/inputs/TextInput/ReadOnlyInput.tsx

```diff
@@ -18,6 +18,7 @@ export const ReadOnlyInput: React.FC<ReadOnlyInputProps> = React.forwardRef<
       backgroundColor: 'transparent',
       width,
       textAlign: number ? 'right' : undefined,
+      color: 'inherit',
```

Comment:
to do the row as a different color when invalid I added a bunch of inherits and applied the color to the row.. hopefully didn't break anything else 😁